/**:
  ros__parameters:
    CONTROL_MODE: actuation_cmd # accel_input, actuation_cmd, external_accel_input, external_actuation_cmd
    LOAD_ROSBAG2_FILES: false
    accel_map_path: /home/kogure/pilot-auto.xx1/src/autoware/universe/vehicle/autoware_raw_vehicle_cmd_converter/data/default/accel_map.csv
    brake_map_path: /home/kogure/pilot-auto.xx1/src/autoware/universe/vehicle/autoware_raw_vehicle_cmd_converter/data/default/brake_map.csv

    # COURSE_NAME: eight_course
    # COURSE_NAME: u_shaped_return
    # COURSE_NAME: straight_line_positive
    # COURSE_NAME: straight_line_negative
    COURSE_NAME: reversal_loop_circle
    # COURSE_NAME: along_road

    NUM_BINS_V: 10
    NUM_BINS_STEER: 20
    NUM_BINS_A: 10
    NUM_BINS_ABS_STEER_RATE: 5
    NUM_BINS_JERK: 10
    NUM_BINS_ACCEL_PEDAL_INPUT: 10
    NUM_BINS_BRAKE_PEDAL_INPUT: 12
    V_MIN: 0.0
    V_MAX: 11.5
    STEER_MIN: -0.6
    STEER_MAX: 0.6
    A_MIN: -1.0
    A_MAX: 1.0
    ABS_STEER_RATE_MIN: 0.0
    ABS_STEER_RATE_MAX: 0.3
    JERK_MIN: -0.5
    JERK_MAX: 0.5
    ACCEL_PEDAL_INPUT_MAX: 0.3
    ACCEL_PEDAL_INPUT_MIN: 0.01
    BRAKE_PEDAL_INPUT_MAX: 0.4
    BRAKE_PEDAL_INPUT_MIN: 0.0

    MASK_NAME: default
    VEL_ACC_THRESHOLD: 40
    VEL_STEER_THRESHOLD: 20
    VEL_ABS_STEER_RATE_THRESHOLD: 20

    max_lateral_accel: 2.00
    lateral_error_threshold: 1.50
    yaw_error_threshold: 0.75
    velocity_limit_by_tracking_error: 1.0
    mov_ave_window: 50
    target_longitudinal_velocity: 6.0

    pure_pursuit_type: linearized
    # pure_pursuit_type: naive
    wheel_base: 2.79
    acc_kp: 1.0
    lookahead_time: 2.0
    min_lookahead: 2.0
    linearized_pure_pursuit_steer_kp_param: 2.0
    linearized_pure_pursuit_steer_kd_param: 2.0
    stop_acc: -2.0
    stop_jerk_lim: 5.0
    lon_acc_lim: 1.5
    lon_jerk_lim: 5.0
    steer_lim: 0.6
    steer_rate_lim: 0.6
